crates/hir-def/src/resolver.rs

```diff
@@ -479,6 +479,10 @@ impl Resolver {
         res.map
     }
 
+    pub fn extern_crates_in_scope(&self) -> Vec<Name> {
+        self.module_scope.def_map.extern_prelude().map(|(name, _)| name.clone()).collect()
```

Comment:
Should be no need to collect here and we are also interested in the module here
```suggestion
    pub fn extern_crates_in_scope(&self) -> impl Iterator<Item = (Name, ModuleId)> {
        self.module_scope.def_map.extern_prelude().map(|(name, mid)| (name.clone(), mid))
```